### ISSUE TITLE
Still reproduce bug with static field option on a Super Table

### ISSUE THREAD

**Issue by asparkam** [Firstborn/Craft-CMS-Migration-Manager#15]
Still reproduces [this bug](https://github.com/Firstborn/Craft-CMS-Migration-Manager/issues/12). When I run Create migration of entry and it has SuperTable field with the Static field option enabled (when disabled it's working), I have this error message: 
![image](https://user-images.githubusercontent.com/30487905/34915975-88034878-f962-11e7-88d5-a7d941e7d386.png)

Log trace: `Call to a member function getFieldLayout() on null in /sitepath/craft/plugins/migrationmanager/services/MigrationManager_BaseContentMigrationService.php:9`

**Comment by fb-derrick-grigg** [comment]
@asparkam I have tested this with normal and static supertables. In both cases the field and content migrations are working fine. Which version of Supertable and MigrationManager are you using?

**Comment by asparkam** [comment]
@fb-derrick-grigg Super Table 1.0.6, Migration Manager 1.0.8

**Comment by asparkam** [comment]
@fb-derrick-grigg try to add Matrix field inside SupertTable. I paid attention for it just now. I think the problem with the levels of nesting. Steps: 
1) Create new Supertable field without settings - all is default and just enable static option. 
2) Create new Matrix inside SuperTable field. 
3) Create new single Section and add SuperTable field to it.
4) Try to make migration if successfully - add some content inside and try one more time. Also you can try to change the settings of the field to Asset or to play with required and tranlated fields.

Also I reproduced it with same variation of Neo field with nesting level: Section->Neo->Supertable->Matrix

**Comment by asparkam** [comment]
Full stack trace:
2018/01/15 17:44:46 [error] [exception.Error] Error: Call to a member function getFieldLayout() on null in sitepath\craft\plugins\migrationmanager\services\MigrationManager_BaseContentMigrationService.php:9
Stack trace:
#0 sitepath\craft\plugins\migrationmanager\services\MigrationManager_BaseContentMigrationService.php(79): Craft\MigrationManager_BaseContentMigrationService->getContent(Array, NULL)
#1 sitepath\craft\plugins\migrationmanager\services\MigrationManager_BaseContentMigrationService.php(10): Craft\MigrationManager_BaseContentMigrationService->getFieldContent(Array, Object(Craft\FieldLayoutFieldModel), Object(Craft\EntryModel))
#2 sitepath\craft\plugins\migrationmanager\services\MigrationManager_EntriesContentService.php(46): Craft\MigrationManager_BaseContentMigrationService->getContent(Array, Object(Craft\EntryModel))
#3 sitepath\craft\plugins\migrationmanager\services\MigrationManager_BaseMigrationService.php(67): Craft\MigrationManager_EntriesContentService->exportItem('4660', true)
#4 sitepath\craft\plugins\migrationmanager\services\MigrationManager_MigrationsService.php(141): Craft\MigrationManager_BaseMigrationService->export(Array)
#5 sitepath\craft\plugins\migrationmanager\actions\MigrationManager_MigrateEntryElementAction.php(17): Craft\MigrationManager_MigrationsService->createContentMigration(Array)
#6 sitepath\craft\app\controllers\ElementIndexController.php(192): Craft\MigrationManager_MigrateEntryElementAction->performAction(Object(Craft\ElementCriteriaModel))
#7 sitepath\craft\app\framework\web\actions\CInlineAction.php(49): Craft\ElementIndexController->actionPerformAction()
#8 sitepath\craft\app\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 sitepath\craft\app\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 sitepath\craft\app\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 sitepath\craft\app\framework\web\CWebApplication.php(282): CController->run('performAction')
#12 sitepath\craft\app\etc\web\WebApp.php(823): CWebApplication->runController('elementIndex/pe...')
#13 sitepath\craft\app\etc\web\WebApp.php(287): Craft\WebApp->_processActionRequest()
#14 sitepath\craft\app\framework\base\CApplication.php(185): Craft\WebApp->processRequest()
#15 sitepath\craft\app\index.php(62): CApplication->run()
#16 sitepath\_index.php(16): require_once('path\to\sit...')
#17 sitepath\index.php(32): require_once('path\to\sit...')
#18 {main}
REQUEST_URI=/index.php/admin/actions/elementIndex/performAction
HTTP_REFERER=http://sitename.dev/admin/entries/singles

**Comment by asparkam** [comment]
Full stack trace:
```
2018/01/15 17:44:46 [error] [exception.Error] Error: Call to a member function getFieldLayout() on null in sitepath\craft\plugins\migrationmanager\services\MigrationManager_BaseContentMigrationService.php:9
Stack trace:
#0 sitepath\craft\plugins\migrationmanager\services\MigrationManager_BaseContentMigrationService.php(79): Craft\MigrationManager_BaseContentMigrationService->getContent(Array, NULL)
#1 sitepath\craft\plugins\migrationmanager\services\MigrationManager_BaseContentMigrationService.php(10): Craft\MigrationManager_BaseContentMigrationService->getFieldContent(Array, Object(Craft\FieldLayoutFieldModel), Object(Craft\EntryModel))
#2 sitepath\craft\plugins\migrationmanager\services\MigrationManager_EntriesContentService.php(46): Craft\MigrationManager_BaseContentMigrationService->getContent(Array, Object(Craft\EntryModel))
#3 sitepath\craft\plugins\migrationmanager\services\MigrationManager_BaseMigrationService.php(67): Craft\MigrationManager_EntriesContentService->exportItem('4660', true)
#4 sitepath\craft\plugins\migrationmanager\services\MigrationManager_MigrationsService.php(141): Craft\MigrationManager_BaseMigrationService->export(Array)
#5 sitepath\craft\plugins\migrationmanager\actions\MigrationManager_MigrateEntryElementAction.php(17): Craft\MigrationManager_MigrationsService->createContentMigration(Array)
#6 sitepath\craft\app\controllers\ElementIndexController.php(192): Craft\MigrationManager_MigrateEntryElementAction->performAction(Object(Craft\ElementCriteriaModel))
#7 sitepath\craft\app\framework\web\actions\CInlineAction.php(49): Craft\ElementIndexController->actionPerformAction()
#8 sitepath\craft\app\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 sitepath\craft\app\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 sitepath\craft\app\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 sitepath\craft\app\framework\web\CWebApplication.php(282): CController->run('performAction')
#12 sitepath\craft\app\etc\web\WebApp.php(823): CWebApplication->runController('elementIndex/pe...')
#13 sitepath\craft\app\etc\web\WebApp.php(287): Craft\WebApp->_processActionRequest()
#14 sitepath\craft\app\framework\base\CApplication.php(185): Craft\WebApp->processRequest()
#15 sitepath\craft\app\index.php(62): CApplication->run()
#16 sitepath\_index.php(16): require_once('path\to\sit...')
#17 sitepath\index.php(32): require_once('path\to\sit...')
#18 {main}
REQUEST_URI=/index.php/admin/actions/elementIndex/performAction
HTTP_REFERER=http://sitename.dev/admin/entries/singles
```

**Comment by fb-derrick-grigg** [comment]
Is the entry you are attempting to migrate localized?

**Comment by asparkam** [comment]
@fb-derrick-grigg site has locales, but entry isn't localized

**Comment by fb-derrick-grigg** [comment]
@asparkam try the latest release. It looks like it was attempting to create migrations for localized entries, even if the entry had not been created.

**Comment by asparkam** [comment]
@fb-derrick-grigg ok, I will try it tomorrow and write here about results.

**Comment by asparkam** [comment]
@fb-derrick-grigg nope, 1.0.8.2 release is not helped. Still reproduced the same. May be it's [related problem](https://github.com/Firstborn/Craft-CMS-Migration-Manager/issues/16)? Because this problem is active too for me.

**Comment by fb-derrick-grigg** [comment]
I am working on a solution. The relations get difficult to manage with nested content. It's having issues identifying the correct nested supertable block to assign the content to when running the migration.

**Comment by fb-derrick-grigg** [comment]
try the latest release when you have a chance

**Comment by asparkam** [comment]
@fb-derrick-grigg [MigrationManagerHelper.php](https://github.com/Firstborn/Craft-CMS-Migration-Manager/blob/v1.0.8.3/migrationmanager/helpers/MigrationManagerHelper.php) file has not method `getNeoBlockType()` and error occurring during migration:
`2018/01/18 05:27:49 [error] [exception.Error] Error: Call to undefined method Craft\MigrationManagerHelper::getNeoBlockType() in sitepath\craft\plugins\migrationmanager\services\MigrationManager_BaseContentMigrationService.php:149`

**Comment by fb-derrick-grigg** [comment]
Sorry, missed one update, try the latest.

**Comment by asparkam** [comment]
I just checked both problems on release 1.0.8.5. [This problem](https://github.com/Firstborn/Craft-CMS-Migration-Manager/issues/16) is gone (you are wizard, thank you so much!), but there was a small bug:  I can't templating this fields i.e. fields is undefined in template and I can't access them. But after reSaving this entity everything becomes normal. 

But bug with static field option (issue theme) is repeating - steps for reproducing above - is actual.

**Comment by fb-derrick-grigg** [comment]
@asparkam the static field issue for nested SuperTable fields has been fixed in 1.0.8.8
